src/main/java/seedu/duke/common/Messages.java

```diff
@@ -24,7 +24,15 @@
     public static final String MESSAGE_INVALID_COMMAND = "Invalid command. Enter \"help\" for list of valid commands.";
 
     public static final String MESSAGE_SEPARATE_INDICES = NEWLINE + "Separate indices with a blank space.";
+    public static final String MESSAGE_ENTER_INDICES =
+            NEWLINE + "Please enter the indices of the %ss you would like to %s." + MESSAGE_SEPARATE_INDICES;
```

Comment:
Most output looks standardised now to me except for DeleteLessonCommand. I think can standardise it/edit/update UG